ldap_reader/vendor.py

```diff
@@ -100,3 +103,14 @@ def check_enabled(config, attrs_dict):
     '''
     server_type = config["server_type"]
     return _SERVER_ENABLE_ATTR_MAP[server_type][1](attrs_dict)
+
+
+def fix_guid(config, guid):
```

Comment:
Plenty good enough structure for now.

Have we verified that RHDS and OpenLDAP don't throw a binary blob at us? This is why the function exists for AD, and our existing RHDS field implementation just specifies `uid` for this field.